lib/index.ts

```diff
@@ -17,12 +17,16 @@ import {
 } from './backburner/utils';
 
 import searchTimer from './backburner/binary-search';
+
 import DeferredActionQueues from './backburner/deferred-action-queues';
+export { DeferredActionQueues };
```

Comment:
Should this be `export type { DeferredActionQueues }`? I don't think we actually want to export the value, just the type...

Reply:
Probably.